Rakefile

```diff
@@ -55,3 +55,10 @@ desc 'Generate REFERENCE.md'
 task :reference do
   sh 'puppet strings generate --format markdown'
 end
+
+namespace :acceptance do
```

Comment:
if these tests are only for windows, the name should correspond to that somehow, such as windows_acceptance or the like.

Reply:
It is , the rake task is `acceptance:windows` , should I change?

Reply:
ah that's fine, i just couldnt tell by the code